### PR TITLE
[MM-9708] Add load more posts to the bottom of permalink view

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Add load more posts to the bottom of permalink view
- Add load more posts at the bottom whenever there are still more posts to load

I did a few tests and looks good to me but I would like to request for a smoke test in case I missed something obvious.

#### Ticket Link
Jira ticket: [MM-9708](https://mattermost.atlassian.net/browse/MM-9708)

#### Checklist
- [x] Ran `make check-style` to check for style errors (required for all pull requests)
- [x] Ran `make test` to ensure unit and component tests passed
- [x] Has redux changes (https://github.com/mattermost/mattermost-redux/pull/469/commits/2f4df765d377f1052d7f2dc90f45c7aa7412fc87)

### CODE REVIEW

File: components/channel_view/channel_view.jsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ import deferComponentRender from 'components/deferComponentRender';
 import ChannelHeader from 'components/channel_header';
 import CreatePost from 'components/create_post';
 import FileUploadOverlay from 'components/file_upload_overlay.jsx';
-import PostView from 'components/post_view';
+import PostList from 'components/post_view/post_list';
```
Comment (saturninoabril) [inline components/channel_view/channel_view.jsx]: Just change this to PostList - similar to component name - to make it more searchable.

File: components/post_view/post_list/post_list.jsx [hunk 2]
```diff
@@ -12,14 +12,14 @@ import EventTypes from 'utils/event_types.jsx';
...
-import Post from './post';
-import ScrollToBottomArrows from './scroll_to_bottom_arrows.jsx';
-import CreateChannelIntroMessage from './channel_intro_message';
+import CreateChannelIntroMessage from 'components/post_view/channel_intro_message';
+import DateSeparator from 'components/post_view/date_separator.jsx';
+import FloatingTimestamp from 'components/post_view/floating_timestamp.jsx';
+import LoadingScreen from 'components/loading_screen.jsx';
+import NewMessageIndicator from 'components/post_view/new_message_indicator.jsx';
+import Post from 'components/post_view/post';
+import ScrollToBottomArrows from 'components/post_view/scroll_to_bottom_arrows.jsx';
```
Comment (saturninoabril) [inline components/post_view/post_list/post_list.jsx]: Move PostList to its own folder since it's mixed with other common/shared components.

**Comment by jasonblais** [comment]
@saturninoabril Looks like the build failed here

**Comment by jasonblais** [comment]
@saturninoabril Are there any next steps on this one?

**Comment by saturninoabril** [comment]
I'm closing this since this will be covered by the on going Scrolling Overhaul epic.

**Pull request closed without merging**